docs/sources/features/datasources/azuremonitor.md

```diff
@@ -0,0 +1,271 @@
+
+### Formatting Legend Keys with Aliases for the Azure Monitor Service
+
+The default legend formatting for the Azure Monitor API is:
+
+`resourceName{dimensionValue=dimensionName}.metricName`
+
+These can be quite long but this formatting can be changed using aliases. In the Legend Format field, the aliases which are defined below can be combined any way you want.
+
+Azure Monitor Examples:
+
+- `dimension: {{dimensionvalue}}`
+- `{{resourcegroup}} - {{resourcename}}`
+
+### Alias Patterns for Azure Monitor
+
+- `{{resourcegroup}}` = replaced with the value of the Resource Group
+- `{{namespace}}` = replaced with the value of the Namespace (e.g. Microsoft.Compute/virtualMachines)
+- `{{resourcename}}` = replaced with the value of the Resource Name
+- `{{metric}}` = replaced with metric name (e.g. Percentage CPU)
+- `{{dimensionname}}` = replaced with dimension key/label (e.g. blobtype)
+- `{{dimensionvalue}}` = replaced with dimension value (e.g. BlockBlob)
+
+### Templating with Variables for the Azure Monitor Service
+
+Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place. Variables are shown as dropdown select boxes at the top of the dashboard. These dropdowns makes it easy to change the data being displayed in your dashboard.
+
+Note that the Azure Monitor service does not support multiple values yet. If you want to visualize multiple time series (for example, metrics for server1 and server2) then you have to add multiple queries to able to view them on the same graph or in the same table.
+
+The Azure Monitor Datasource Plugin provides the following queries you can specify in the `Query` field in the Variable edit view. They allow you to fill a variable's options list.
+
+| Name                                                     | Description                                                    |
+| -------------------------------------------------------- | -------------------------------------------------------------- |
+| *ResourceGroups()*                                       | Returns a list of resource groups.                             |
+| *Namespaces(aResourceGroup)*                             | Returns a list of namespaces for the specified resource group. |
+| *ResourceNames(aResourceGroup, aNamespace)*              | Returns a list of resource names.                              |
+| *MetricNames(aResourceGroup, aNamespace, aResourceName)* | Returns a list of metric names.                                |
+
+Examples:
+
+- Resource Groups query: `ResourceGroups()`
+- Passing in metric name variable: `Namespaces(cosmo)`
+- Chaining template variables: `ResourceNames($rg, $ns)`
+- Do not quote parameters: `MetricNames(hg, Microsoft.Network/publicIPAddresses, grafanaIP)`
+
+{{< docs-imagebox img="/img/docs/v60/azuremonitor-service-variables.png" class="docs-image--no-shadow" caption="Nested Azure Monitor Template Variables" >}}
```

Comment:
Broken image, but I guess you already know that

Reply:
Not for me - did you download the latest from grafana.org and build the docker container?

Reply:
Double checked that they are pushed and they were: https://github.com/grafana/grafana.org/tree/master/static/img/docs/v60